.github/workflows/pull_request_workflow.yml

```diff
@@ -20,5 +20,12 @@ jobs:
         with:
           java-version: ${{ matrix.java-version }}
           distribution: temurin
+
       - name: Build with Gradle
-        run: ./gradlew build distTar
+        run: ./gradlew build distTar -x test
+
+      - name: Run integration tests
+        run: ./gradlew integrationTest
```

Comment:
So we are skipping the unit tests entirely, right?

Reply:
My bad. Meant to disable integrationTest on previous state. Should be fixed now.